src/app/public/index.ts

```diff
@@ -1 +1,8 @@
-export * from './modules';
+export * from './modules/inline-form/inline-form-adapter.service';
```

Comment:
Do we want to export the adapter service?

Reply:
Nope. Removed.